src/display_image.c

```diff
@@ -0,0 +1,14 @@
+// under the MIT license, by marius851000
+
+#include "display_image.h"
+
+// require the top screen to be configured to LDLC mode, then switched back to BG mode for engine 2
+void displayImageOnConfiguredTopScreen(const char * filepath) {
+    struct file_stream file;
+    DataTransferInit();
+    FileOpen(&file, filepath);
```

Comment:
Need to call [`FileInit`](https://github.com/UsernameFodder/pmdsky-debug/blob/4f361d8590e4ba28306670a20046e1c6ba38fa59/symbols/arm9/libs.yml#L2098-L2108) or [`FileInitVeneer`](https://github.com/UsernameFodder/pmdsky-debug/blob/4f361d8590e4ba28306670a20046e1c6ba38fa59/symbols/arm9.yml#L932-L944) prior to file opening.